sale_order_type/views/sale_order_type_view.xml

```diff
@@ -1,119 +1,114 @@
 <?xml version="1.0" encoding="utf-8"?>
-<openerp>
-    <data>
-
-        <record id="sot_sale_order_type_form_view" model="ir.ui.view">
-            <field name="name">sale.order.type.form.view</field>
-            <field name="model">sale.order.type</field>
-            <field name="arch" type="xml">
-                <form string="Type">
-                    <sheet>
-                        <group>
-                            <div class="oe_title">
-                                <label for="name" class="oe_edit_only"/>
-                                <h1>
-                                    <field name="name"/>
-                                </h1>
-                            </div>
-                        </group>
+<odoo>
+    <record id="sot_sale_order_type_form_view" model="ir.ui.view">
+        <field name="name">sale.order.type.form.view</field>
+        <field name="model">sale.order.type</field>
+        <field name="arch" type="xml">
+            <form string="Type">
+                <sheet>
+                    <group>
+                        <div class="oe_title">
+                            <label for="name" class="oe_edit_only"/>
+                            <h1>
+                                <field name="name"/>
+                            </h1>
+                        </div>
+                    </group>
+                    <group>
                         <group>
-                            <group>
-                                <field name="warehouse_id" />
-                                <field name="sequence_id" />
-                            </group>
-                            <group>
-                                <field name="journal_id" />
-                                <field name="refund_journal_id" />
-                                <field name="picking_policy" />
-                                <field name="order_policy" />
-                                <field name="invoice_state" />
-                            </group>
+                            <field name="warehouse_id" />
+                            <field name="company_id" />
```

Comment:
I think it should be nicer if `company_id` has the attribute `invisible=true` whenever `warehouse_is` is empty.